adaptive_shield/manifest.json

```
@@ -0,0 +1,56 @@
+{
+	"manifest_version": "2.0.0",
+	"app_id": "adaptive-shield",
+	"app_uuid":"0c72bf61-1de6-4408-8a24-86f8e3413e07",
+	"display_on_public_website": true,
+	"tile": {
+      "overview": "README.md#Overview",
+      "configuration": "README.md#Setup",
+      "support": "README.md#Support",
+      "changelog": "CHANGELOG.md",
+      "description": "Track SaaS posture alerts",
+      "title": "Adaptive Shield",
+      "media": [
+        {
+        "caption": "SaaS environments posture in Adaptive Shield",
+        "image_url": "images/posture.png",
+        "media_type": "image"
+        },
+        {
+        "caption": "Security Checks on SaaS configurations in Adaptive Shield",
+        "image_url": "images/security_checks.png",
+        "media_type": "image"
+        }
+      ],
+      "classifier_tags": [
+       "Supported OS::Linux",
+       "Supported OS::Mac OS",
+       "Supported OS::Windows",
+       "Category::Alerting",
+       "Category::Cloud",
+       "Category::Security",
+       "Offering::Integration"
+       ]
+    },
+    "author": {
+      "homepage": "https://www.adaptive-shield.com/",
+      "name": "Adaptive Shield",
+      "support_email": "support@adaptive-shield.com",
+      "sales_email": "info@adaptive-shield.com"
+    },
+    "assets": {
+      "integration": {
+        "source_type_name": "adaptive_shield",
+        "configuration": {},
+        "events": {
+          "creates_events": true
+        }
+      },
+      "dashboards": {
+        "adaptive_shield_overview": "dashboards/adaptive_shield_overview.json"
```

```suggestion
        "adaptive_shield_overview": "assets/dashboards/adaptive_shield_overview.json"
```
This path looks broken